SS14.Admin/Pages/Index.cshtml

```diff
@@ -11,4 +11,5 @@
     <a class="btn btn-primary" asp-page="./Connections/Index">Connections</a>
     <a class="btn btn-primary" asp-page="./Logs/Index">Logs</a>
     <a class="btn btn-primary" asp-page="./Characters/Index">Characters</a>
+    <button type="button" class="btn btn-primary" onclick="toggleDarkMode()">Toggle darkmode</button>

```

Comment:
Can you move this to a User settings page to the left of the Hello "your username here" use somthing like this icon to link the settings page <i class="fas fa-user-cog"></i> I have some more stuff I want to add to a setting page such as the PII censoring mechanic.